src/PointCloudRasterizers.jl

```diff
@@ -74,6 +74,28 @@ function Base.filter!(index::PointCloudIndex, condition=nothing)
 	end
 end
 
+"""Filter out index on given condition relative to a reduced layer."""
+function Base.filter!(index::PointCloudRasterizers.PointCloudIndex, raster::GeoArray, condition=nothing)
+
+    #check that size and affine info match
```

Comment:
I will move this check to GeoArrays eventually, good to have an equal operation for all fields/sizes except for the values in the array.